cli/tests/vanilla/testVotingWithKeyDeactivation.sh

```diff
@@ -38,28 +37,53 @@ $MACI_CLI deactivateKey \
 $MACI_CLI timeTravel \
     --seconds 10
 
-# $MACI_CLI generateNewKey  \ add when implemented
-
 # --from-block since MACI deployed
 $MACI_CLI confirmDeactivation \
     --poll-id $POLL_ID \
-    --privkey macisk.49953af3585856f539d194b46c82f4ed54ec508fb9b882940cbe68bbc57e59e \
+    --privkey macisk.fd7aa614ec4a82716ffc219c24fd7e7b52a2b63b5afb17e81c22fe21515539c \
     --from-block 0 \
     --batch-size 1 \
 
-
 # key deactivation happens after original period 86400s from prepare_test.sh has expired 
 $MACI_CLI timeTravel \
     --seconds 90000
 
 # missing triggering of smart contract code to pass batches
 $MACI_CLI completeDeactivation \
     --poll-id $POLL_ID \
-    --privkey macisk.49953af3585856f539d194b46c82f4ed54ec508fb9b882940cbe68bbc57e59e \
+    --privkey macisk.fd7aa614ec4a82716ffc219c24fd7e7b52a2b63b5afb17e81c22fe21515539c \
     --state-num-sr-queue-ops 1 \
     --deactivated-keys-num-sr-queue-ops 1 \
     --from-block 0 \
     --process-deactivation-witnessgen "$ZKEYS_DIR"/ProcessDeactivationMessages_"$PROCESS_DEACTIVATION_MESSAGES_PARAMS" \
     --process-deactivation-zkey "$ZKEYS_DIR"/ProcessDeactivationMessages_"$PROCESS_DEACTIVATION_MESSAGES_PARAMS".0.zkey \
     --rapidsnark ~/rapidsnark/build/prover \
 
+# TODO: consider using genMaciKeypair instead of hardcoded key pairs
+
+$MACI_CLI generateNewKey \
```

Comment:
Please comment everything below as the test would fail for now because the generate key command is not implemented.